packages/api/src/functions/security/ms-defender-utils.ts

```diff
@@ -0,0 +1,74 @@
+import process from 'node:process';
+import { HttpRequest } from '@azure/functions';
+
+export function getMsDefenderUserJson(request: HttpRequest): string {
```

Comment:
worth to add a link to our documentation somehere

Reply:
added in README file, will also add method description